lib/active_job/queue_adapters/amazon_sqs_adapter.rb

```diff
@@ -4,7 +4,7 @@
 
 module ActiveJob
   module QueueAdapters
-    class AmazonSqsAdapter
+    class AmazonSqsAdapter < AbstractAdapter
```

Comment:
@mullermp https://github.com/rails/rails/blob/4fa56814f18fd3da49c83931fa773caa727d8096/activejob/lib/active_job/queue_adapters/abstract_adapter.rb#L14 see that enqueue method (it was missing and that's why it was blowing up in tests).

I _think_ the request spec tests (when using sqs driver) were actually testing the queue flow fully whereas before it wasn't so this isn't raised on main.

Reply:
no that doesn't make any sense (the test coverage thing), but it does make sense from a rails-main angle

Reply:
Well. Maybe it's the transaction wrapper _and_ the rails main changes. Like it wasn't in a transaction before...now it is. So it caught something that wasn't caught before. I'll verify by enabling transactions on main and seeing if it melts.

Reply:
Well. Maybe it's the transaction wrapper _and_ the rails main change? Like it wasn't in a transaction before/no db stuff...now it is?